docs/decisions/0004-managing-django-settings.rst

```diff
@@ -0,0 +1,37 @@
+Status
+======
+
+Accepted
+
+
+Context
+=======
+
+Settings are confusing right now and there are way too many layers. We have
+multiple python settings file that "inherit" from each other. In some cases some
+layers of the inheritance tree also pull settings from config files leading to
+more confusion.
+
+
+Decision
+========
+Rather than having multiple python settings files that override things
+differently we will move to just a few python files with most of the settings
+variance living in YAML config files that are managed by environment operators.
+
+common.py - This file will house the defaults for all settings that are
```

Comment:
Is this an opportunity to rename `common.py` to a more standard Django name (`settings.py`)?

Reply:
I was considering `defaults.py` to make it more clear as well.  I'd be happy to pull this in as a part of this decision.  Let me know if you prefer `common.py` or `defaults.py`